src/component/Photo/Photo.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import './Photo.css'
+const Photo = () => {
+  return (
+    <div className="imgs">
+      <img src="./../../../img/circle.png" alt="" className="circular"/>
+      <img src="./../../../img/person.png" alt="" className="person" />
+        <img src="./../../../img/react.png" alt="" className="react" /> 
+      <img src="./../../../img/star.png" alt="" className="star"/>
+      <img src="./../../../img/Figma.png " alt="" className="figma"/>
+      <img src="./../../../img/sttar.png " alt="" className="sttar"/>

```

Comment:
This is against the idea of "reusable components".
Please loop over an array that contains the objects with the props as key-value pairs, and return one "img" tag.